reference/Chronicler2.v0.yaml

```diff
@@ -138,102 +277,166 @@ components:
                 type: string
               valid_from:
                 type: string
+                format: date-time
               valid_to:
-                type: 'null'
+                type:
+                  - string
+                  - 'null'
+                format: date-time
               data:
                 type: object
                 properties:
-                  attributeRatings:
-                    type: array
-                    items:
-                      type: object
-                      properties:
-                        category:
-                          type: string
-                        name:
-                          type: string
-                        stars:
-                          type:
-                            - integer
-                            - number
-                  attributes:
+                  awayPitcher:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      name:
+                        type: string
+                  awayTeam:
+                    type: object
+                    properties:
+                      emoji:
+                        type: string
+                      gameWinner:
+                        type: boolean
+                      id:
+                        type: string
+                      name:
+                        type: string
+                      nickname:
+                        type: string
+                      primaryColor:
+                        type: string
+                      secondaryColor:
+                        type: string
+                      shorthand:
+                        type: string
+                      slogan:
+                        type: string
+                  cancelled:
+                    type: boolean
+                  complete:
+                    type: boolean
+                  created:
+                    type: string
+                  day:
+                    type: integer
+                  deleted:
+                    type: boolean
+                  gameEventBatches:
                     type: array
                     items:
                       type: object
                       properties:
-                        name:
+                        batchData:
```

Comment:
i'm not super familiar with yaml -- is there any way of annotating that the type of this is a json string?

Reply:
Yes, it's using a `description`. In this case it's less-so YAML, and moreso the OpenAPI 3.1 specification which determines the format and design of the API spec.